### ISSUE TITLE
Docs:  Homepage Accordion Image does not load on Safari

### ISSUE THREAD

**Issue by MateoWartelle** [themesberg/flowbite-react#1128]
- [X] I have searched the [Issues](https://github.com/themesberg/flowbite-react/issues) to see if this bug has already been reported
- [X] I have tested the latest version

## Steps to reproduce

1. Go to https://www.flowbite-react.com/
2. React UI Components

## Current behavior

Describe what is currently happening and why it's a problem.

<img width="1007" alt="accordion-safari" src="https://github.com/themesberg/flowbite-react/assets/27743176/518f301c-3539-4e10-8ade-a4fcdf428068">

## Expected behavior

Describe what you expected to happen.

## Context

What are you trying to accomplish? Does this only happen on a specific browser, screen size, or operating system?

If possible, provide a live example URL, screenshot, video, or a repository with the minimal reproduction of the issue.

**Comment by tulup-conner** [comment]
Yeah I can confirm this is happening but I am perplexed why. Maybe someone else can look into this?
